ena-submission/README.md

```diff
@@ -14,34 +16,80 @@ The ENA submission pod creates a new schema in the loculus DB, this is managed b
 ```
 
 If you want to test the docker image locally. It can be built and run using the commands:
 
-```
+```sh
 docker build -t ena-submission-flyway .
 docker run -it -e FLYWAY_URL=jdbc:postgresql://127.0.0.1:5432/loculus -e FLYWAY_USER=postgres -e FLYWAY_PASSWORD=unsecure ena-submission-flyway flyway migrate
 ```
 
+### Setting up micromamba environment
+
+<details>
+
+<summary> Setting up micromamba </summary>
+
 The rest of the ena-submission pod uses micromamba:
 
-```bash
+```sh
 brew install micromamba
 micromamba shell init --shell zsh --root-prefix=~/micromamba
 source ~/.zshrc
 ```
 
+</details>
+
 Then activate the loculus-ena-submission environment
 
-```bash
-micromamba create -f environment.yml --platform osx-64 --rc-file .mambarc
+```sh
+micromamba create -f environment.yml --rc-file .mambarc
 micromamba activate loculus-ena-submission
 ```
 
+### Using ENA's webin-cli
+
+In order to submit assemblies you will also need to install ENA's `webin-cli.jar`. Their [webpage](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/webin-cli.html) offers more instructions. This pipeline has been tested with `WEBIN_CLI_VERSION=7.3.1`.
+
+```sh
+wget -q "https://github.com/enasequence/webin-cli/releases/download/${WEBIN_CLI_VERSION}/webin-cli-${WEBIN_CLI_VERSION}.jar" -O /package/webin-cli.jar
+```
+
+### Running snakemake
+
 Then run snakemake using `snakemake` or `snakemake {rule}`.
+
+### Run tests
+
+```sh
+micromamba activate loculus-ena-submission
+python3 scripts/test_ena_submission.py
+```
+
+### Testing submission locally
+
+ENA-submission currently is only triggered after manual approval.
+
+The `get_ena_submission_list` runs as a cron-job. It queries Loculus for new sequences to submit to ENA (these are sequences that are in state OPEN, were not submitted by the INSDC_INGEST_USER, do not include ena external_metadata fields and are not yet in the submission_table of the ena-submission schema). If it finds new sequences it sends a notification to slack with all sequences.
+
+It is then the reviewer's turn to review these sequences. [TODO: define review criteria] If these sequences meet our criteria they should be uploaded to [pathoplexus/ena-submission](https://github.com/pathoplexus/ena-submission/blob/main/approved/approved_ena_submission_list.json) (currently we read data from the [test folder](https://github.com/pathoplexus/ena-submission/blob/main/test/approved_ena_submission_list.json) - but this will be changed to the `approved` folder in production). The `trigger_submission_to_ena` rule is constantly checking this folder for new sequences and adding them to the submission_table if they do not exist. Note we cannot yet handle revisions so these should not be added to the approved list [TODO: do not allow submission of revised sequences in `trigger_submission_to_ena`]- revisions will still have to be performed manually.
```

Comment:
```suggestion
It is then the reviewer's turn to review these sequences. [TODO: define review criteria] If these sequences meet our criteria they should be uploaded to [pathoplexus/ena-submission](https://github.com/pathoplexus/ena-submission/blob/main/approved/approved_ena_submission_list.json) (currently we read data from the [test folder](https://github.com/pathoplexus/ena-submission/blob/main/test/approved_ena_submission_list.json) - but this will be changed to the `approved` folder in production). The `trigger_submission_to_ena` rule is constantly checking this folder for new sequences and adding them to the submission_table if they are not already there. Note we cannot yet handle revisions so these should not be added to the approved list [TODO: do not allow submission of revised sequences in `trigger_submission_to_ena`]- revisions will still have to be performed manually.
```

Reply:
Can we raise an Issue for the second TODO before merging this

Reply:
I added a check for this in the code to prevent us from accidentally submitting revisions :-) there is already an issue on figuring out how to handle revisions to ENA: https://github.com/loculus-project/loculus/issues/2428